### PR TITLE
Amend User Stories in DG

### LINKED ISSUE #283: Amend User Stories in Appendix of Developer Guide

### PR DESCRIPTION
**Pull request by PresleyChew**
Fixes #283

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -283,20 +283,23 @@ _{Explain here how the data archiving feature will be implemented}_
...
-| `*`      | busy doctor           | view all my urgent cases                     | attend to those with urgent needs first               |
-| `*`      | doctor                | see a patient’s allergy information          | avoid prescribing harmful medications                 |
+| Priority | As a …​           | I want to …​                                                                | So that I can…​                                        |
+|----------|-------------------|-----------------------------------------------------------------------------|--------------------------------------------------------|
+| `* * *`  | organised doctor  | add new records                                                             | keep track of my existing patients' details            |
+| `* * *`  | organised doctor  | delete records                                                              | remove entries of patients no longer existing          |
+| `* * *`  | organised doctor  | edit records                                                                | amend outdated information in the patients' record     |
```
Comment (hweinian) [inline docs/DeveloperGuide.md:290]: would this be for normal doctor? cause any doctor would probably need to add/delete/edit records

Comment (PresleyChew) [reply docs/DeveloperGuide.md:290]: True! Amended!

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -283,20 +283,23 @@ _{Explain here how the data archiving feature will be implemented}_
...
+| `* * *`  | doctor            | edit records                                                                | amend outdated information in the patients' record     |
+| `* * *`  | busy doctor       | search for a patient by name                                                | quickly access their records                           |
+| `* * *`  | busy doctor       | search for a patient by NRIC                                                | quickly access their records                           |
+| `* * *`  | doctor            | schedule an appointment with a patient                                      | manage my daily workload effectively                   |
+| `* * *`  | doctor            | delete an appointment with a patient                                        | cancel an appointment                                  |
+| `* * *`  | doctor            | list all records                                                            | look through all contacts                              |
+| `* *`    | doctor            | view all my appointments                                                    | know the appointments I have on a certain day          |
```
Comment (TechRaj) [inline docs/DeveloperGuide.md:296]: Do we not need adjectives for doctor?

**Review by TechRaj — COMMENTED** [review]
LGTM. Just a nit I wanna clarify

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (PresleyChew) [reply docs/DeveloperGuide.md:296]: No

**Review by yong-zhuo — APPROVED** [review]

**Pull request merged**